recipe/meta.yaml

```diff
@@ -29,6 +29,10 @@ requirements:
     - python-confluent-kafka  # [not win]
     - redis-py
     - slack-sdk
+    - globus-sdk
+    - appdirs
```

Comment:
This is a dependency package of [`nslsii`](https://github.com/nsls-ii-forge/nslsii-feedstock/blob/b8716ad0991d6860d31f04436d578572d57584f6/recipe/meta.yaml#L24), which we already have here.